source/cloud/azure/azureml.md

```diff
@@ -16,52 +16,59 @@ Follow these high-level steps to get started:
 
 **2. Workspace.** Within the Resource Group, create an Azure Machine Learning service Workspace.
 
-**3. Config.** Within the Workspace, download the `config.json` file, as you will load the details to initialize workspace for running ML training jobs from within your notebook.
-
-![Screenshot of download config file](../../images/azureml-download-config-file.png)
```

Comment:
AzureML puts this config file into JupyterLab's filesystem at `${JUPYTER_SERVER_ROOT}/config.json`... so we can remove this manual step to simplify things a bit 🎉